app/components/autocomplete/slash_suggestion/slash_suggestion_item.tsx

```diff
@@ -0,0 +1,171 @@
+
+const slashIcon = require('@assets/images/autocomplete/slash_command.png');
+const bangIcon = require('@assets/images/autocomplete/slash_command_error.png');
+
+const getStyleFromTheme = makeStyleSheetFromTheme((theme: Theme) => {
+    return {
+        icon: {
+            fontSize: 24,
+            backgroundColor: changeOpacity(theme.centerChannelColor, 0.08),
+            width: 35,
+            height: 35,
+            marginRight: 12,
+            borderRadius: 4,
+            justifyContent: 'center',
+            alignItems: 'center',
+            marginTop: 8,
+        },
+        uriIcon: {
+            width: 16,
+            height: 16,
+        },
+        iconColor: {
+            tintColor: theme.centerChannelColor,
+        },
+        container: {
+            flexDirection: 'row',
+            alignItems: 'center',
+            paddingBottom: 8,
+            paddingHorizontal: 16,
+            overflow: 'hidden',
+        },
+        suggestionContainer: {
+            flex: 1,
+        },
+        suggestionDescription: {
+            fontSize: 12,
+            color: changeOpacity(theme.centerChannelColor, 0.56),
+        },
+        suggestionName: {
+            fontSize: 15,
+            color: theme.centerChannelColor,
+            marginBottom: 4,
+        },
+    };
+});
+
+type Props = {
+    complete: string;
+    description: string;
+    hint: string;
+    onPress: (complete: string) => void;
+    suggestion: string;
+    icon: string;
+}
+
+const SlashSuggestionItem = ({
+    complete = '',
+    description,
+    hint,
+    onPress,
+    suggestion,
+    icon,
+}: Props) => {
+    const insets = useSafeAreaInsets();
+    const theme = useTheme();
+    const style = getStyleFromTheme(theme);
+
+    const completeSuggestion = () => {
+        onPress(complete);
+    };
+
+    let suggestionText = suggestion;
+    if (suggestionText?.[0] === '/' && complete.split(' ').length === 1) {
+        suggestionText = suggestionText.substring(1);
+    }
+
+    if (hint) {
+        if (suggestionText?.length) {
+            suggestionText += ` ${hint}`;
+        } else {
+            suggestionText = hint;
+        }
+    }
+
+    let image = (
+        <Image
+            style={style.iconColor}
+            width={10}
+            height={16}
+            source={slashIcon}
+        />
+    );
+    if (icon === COMMAND_SUGGESTION_ERROR) {
+        image = (
+            <Image
+                style={style.iconColor}
+                width={10}
+                height={16}
+                source={bangIcon}
+            />
+        );
+    } else if (icon.startsWith('http')) {
+        image = (
+            <FastImage
+                source={{uri: icon}}
+                style={style.uriIcon}
+            />
+        );
+    } else if (icon.startsWith('data:')) {
+        if (icon.startsWith('data:image/svg+xml')) {
+            // TODO: What base64 library should we use? Security implications on doing things like this?
```

Comment:
We used to use a base64 library on V1 that is not yet installed in V2. Just wanted to make sure if we want to install that one, install another one (or use something in one already installed), or if we want to handle this differently.

Also, since this information comes from the plugin, it may be considered "unsafe string" and we may have to do some extra checks here (or trust SvgXml to do the checks for us).

Reply:
I believe we have `base64-js` already in `node_modules` does that one work for you?

Reply:
After looking a bit deeper, in theory won't be needed. There is a `atob` function deprecated, and it recommend to use `Buffer.from`. I am using that in the latest commit. Hopefully is good enough.